styles/goldleaf/layout.s2

```diff
@@ -0,0 +1,2500 @@
+                 content: url($*image_entry_tag);
+             } 
+
+             .tag-text{
+                 display:none;
+             } 
+
+             .metadata-label{
+                 display:none;
+             }
+
+             #metadata-mood:before{
+                 content:url($*image_metadata_mood);
+                 padding-right:2px;
+             }
+
+             .metadata-item-location:before{
+                 content:url($*image_metadata_location);
+                 padding-right:2px;
+             }
+
+             .metadata-item-music:before {
+                 content:url($*image_metadata_music);
+                 padding-right:2px;
+             }
+
+             .metadata-item-groups:before {
+                 content:url($*image_metadata_groups);
+                 padding-right:2px;
+             }
+
+             .metadata-item-xpost:before {
+                 content:url($*image_metadata_xpost);
+                 padding-right:2px;
+             }
+             """;
+         }
+
+    var string controlstrip_css = "";
+         if ($*custom_control_strip_colors == "on_gradient" or $*custom_control_strip_colors == "on_no_gradient") {
+         $controlstrip_css = """
+             #lj_controlstrip td { 
+                 border:none;
+             }
+
+             #lj_controlstrip input{
+                 background:$*control_strip_bgcolor;
+                 border:1px solid $*control_strip_bordercolor;
+                 font-weight:bold;
+                 color:$*control_strip_fgcolor;
+             }
+
+             #lj_controlstrip_userpic img{
+                 border-radius:50%;
+                -webkit-border-radius:50%;
+             }
+             """;
+         }
+
+
+    print_custom_control_strip_css();
+
+    """
+
+/*--------------- general ---------------*/
+
+H1, H2, H3 {
+    margin: .25em 0;
+    padding: .25em 0;
+}
+
+img {
+    border: none;
+}
+
+hr {
+    display: none;
+}
+
+.entry-content hr,
+.comment-content hr {
+    display: block;
+    margin: 1em 10%;
+}
+
+body {
+    $page_font
+    margin: 0; 
+    padding: 0;
+    $page_background
+    $page_colors
+}
+
+html body {
+    margin-left: $*margins_size$*margins_unit;
+    margin-right: $*margins_size$*margins_unit;
+}
+
+q { font-style: italic; }
+
+#content-footer {
+    height: 0;
+    font-size: 0; /*to make IE happy*/
+    clear: both;
+}
+
+#header, #footer {
+    margin: 0;
+    clear: both;
+}
+
+blockquote{
+    font-style:italic;
+}
+
+blockquote:first-letter{
+    font-size:3.6em;
+    float:left;
+    padding:0 .1em 0 0;
+    line-height:.5;
+}
+
+#canvas{
+    $canvas_background
+}
+
+#primary, #secondary, #tertiary { width: 100%; }
+
+/*--------------- header ---------------*/
+
+#header {
+    $header_background
+    $page_title_colors
+    font-style:italic;
+}
+
+#header .inner{
+    padding:.25em 1.5em;
+}
+
+@media $medium_media_query {
+    #header .inner{
+        width: $*topbar_width_size$*topbar_width_units;
```

Comment:
This evaluates to "width: ;" on my dreamhack, which means $*topbar_width_size and $*topbar_width_units are blank somehow?  Now that I look again at the same layer on DW that you linked me to earlier, it looks like it had the same problem there as well.